### PR TITLE
Add Caikit template details to README

### PR DESCRIPTION
**Pull request by hickeyma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The model is served by a [gRPC](https://grpc.io) server which can run as is or i
 
 There is an example of a client [here](https://github.com/caikit/caikit/blob/main/examples/text-sentiment/client.py) which is a simple Python CLI which calls the model and queries it for sentiment analysis on 2 different pieces of text. The client also references the module configuration.
 
-Check out the full [Text Sentiment example](examples/text-sentiment/) to understand how to load and infer a model using Caikit.
+Check out the full [Text Sentiment example](examples/text-sentiment/) to understand how to load and infer a model using Caikit. There is also a [GitHub template](https://github.com/caikit/caikit-template) available which if you click on the green `Use this template` button generates a repository in your GitHub account with a simple customized model which is wrapped to be served by the Caikit runtime.
```
Comment (gkumbhat) [inline README.md:44]: Should we rephrase it to:
```
if you want to get started with developing and integrating your algorithm using `caikit` framework, checkout [GitHub template](https://github.com/caikit/caikit-template), which if you click on the green `Use this template` button generates a repository in your GitHub account with a simple customized module which is wrapped to be served by the Caikit runtime. This template is designed to be extended for module implementations.
```

Comment (hickeyma) [reply README.md]: Like the suggestions @gkumbhat, thanks.

**Comment by hickeyma** [comment]
Thanks for the review @gkumbhat. Updated and ready for review again.

**Review by gkumbhat — APPROVED** [review]
